test/evals/eval_data.yaml

```diff
@@ -85,19 +86,20 @@
 
 - conversation_group: mno_cluster_workflow_conv
   description: Create multi-node cluster and then retrieve ISO in two steps
+  cleanup_script: <workdir-placeholder>/delete_cluster.sh
   conversation:
     - eval_id: create_eval_test_multinode
       eval_types: [tool_eval, response_eval:accuracy, response_eval:sub-string]
-      eval_query: Create a multi-node cluster named 'eval-test-multinode-uniq-cluster-name' with OpenShift 4.18.22 and domain test.local
+      eval_query: Create a multi-node cluster named 'eval-test-multinode-uniq-cluster-name' with OpenShift 4.18.22 and domain test.local.
       expected_tool_calls:
         - - tool_name: create_cluster
             arguments:
               name: "eval-test-multinode-uniq-cluster-name"
               version: "4\\.18\\.22"
               base_domain: "test\\.local"
               single_node: "(?i:false)"
-              cpu_architecture: None
-              ssh_public_key: None
+              cpu_architecture: "x86_64"
```

Comment:
Why not add x86_64 to the prompt to make this more reliable since it doesn't actually matter if it's provided or not?